modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc

```diff
@@ -41,11 +41,11 @@ Create a new directory on your system and copy the file to the new directory:
 [source, console]
 ----
 mkdir ~/prometheus
-cp couchbase_sd_config_my-cluster.yml ~/prometheus
+cp couchbase_sd_config_my-cluster.yaml ~/prometheus
 ----
 
 == Create the Prometheus configuration file
-Create a new file called `prometheus.yml` in the directory containing the targets file. 
+Create a new file called `prometheus.yml` in the directory containing the targets file.
 Populate the file with the following text:
 
 .prometheus.yml
```

Comment:
Strangely enough … no. 😕

The file name has to be `prometheus.yml`; I tried it with `prometheus.yaml` and the docker run failed with a `couldn't find prometheus.yml` error.

What I'll do is submit a change request so that the script will accept both variations of the extension.